docs/seed-image-generation.md

```diff
@@ -0,0 +1,141 @@
+  - Same ACM/MCE version
+    - Exception: Hub for seed SNO must not have extra ACM addons enabled (ie. observability). The LCA orchestration confirms that no such addons are present on the seed SNO as part of its system config validation.
+- OADP operator must be deployed
+- Container storage must be setup as shared between stateroots, either via separate partition or the bind-mount mechanism (*TODO*: provide examples of both?)
+- Required dnsmasq configuration (*TODO*: This will ultimately be provided via ACM, but override mechanism until such time as its available. Provide example of generating lab-specific site-policy or machine-config). The LCA orchestration confirms that this required configuration is present on the seed SNO as part of its system config validation.
+
+## SeedGenerator CR
+
+The Lifecycle Agent provides orchestration of the IBU Seed Image generation, triggered by creating a `SeedGenerator` CR. Additionally, a `seedgen` `Secret` is required to provide the auth necessary for publishing the seed image, as well as the optional `hubKubeconfig` if the LCA is to handle the `ManagedCluster`.
+
+*TODO*: Provide helper script for generating the `seedgen` `Secret` and `seedimage` `SeedGenerator` CRs
+
+### Creating the seedgen Secret CR
+
+The `seedgen` `Secret`, created in the `openshift-lifecycle-agent` namespace, allows the user to provide the following information:
+
+- `seedAuth`: base64-encoded auth file for write-access to the registry for pushing the generated seed image
+- `hubKubeconfig`: (Optional) base64-encoded kubeconfig for admin access to the hub, in order to deregister the seed
+  cluster from ACM. If this is not present in the secret, the ACM cleanup will be skipped.
+
+[!IMPORTANT] This `Secret` must be named `seedgen` and must be created in the `openshift-lifecycle-agent` namespace.
+
+The `seedAuth` is the base64-encoded auth file containing credentials with write-access to the registry to which the seed image is to be published. The auth file itself can be created with limited credentials by running a `podman login` command, such as:
+
+```console
+# MY_USER=myuserid
+# AUTHFILE=/tmp/my-auth.json
+# podman login --authfile ${AUTHFILE} -u ${MY_USER} quay.io/${MY_USER}
+Password:
+Login Succeeded!
+# base64 -w 0 ${AUTHFILE} ; echo
+ewoJImF1dGhzIjogewoJCSJxdWF5LmlvL215dXNlcmlkIjogewoJCQkiYXV0aCI6ICJub3R0aGVyZWFsYXV0aHN0cmluZyIKCQl9Cgl9Cn0K
+#
+```
+
+Example:
+
+```yaml
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: seedgen
+  namespace: openshift-lifecycle-agent
+type: Opaque
+data:
+  seedAuth: <encoded authfile>
+  hubKubeconfig: <encoded kubeconfig>
+```
+
+### Creating the seedimage SeedGenerator CR
+
+The `seedimage` `SeedGenerator` CR allows the user to provide the following information:
+
+- `seedImage`: The pullspec (ie. registry/repo:tag) for the generated image
+- `recertImage`: (Optional) Allows user to specify the recert tool image to pass to ibu-imager
```

Comment:
Since this field is really dev-only and this doc is geared towards end-user, I'll drop it altogether from the doc. It is still referenced in HACKING.md